docs/src/user-guide/configuring/language-options.md

```diff
@@ -9,6 +9,8 @@ eleventyNavigation:
 
 ---
 
+The JavaScript ecosystem has a variety of runtimes, versions, extensions and frameworks. Each of these can have different supported syntax and global variables. ESLint lets you configure language options specific to the JavaScript used in your project. You can also add custom global variables and plugins to extend ESLint to support your project's language options.
```

Comment:
"You can also add custom global variables" might be confusing, because configuring language options includes adding globals, so they're already included in the previous sentence "ESLint lets you configure language options specific to the JavaScript used in your project", while this sounds like adding globals is something separate from language options.

Reply:
addressed in [ff67680](https://github.com/eslint/eslint/pull/16511/commits/ff676800b30868810cf865becac69f2f2f169c5a)